Index: src/Entity/Client.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- src/Entity/Client.php	(revision 938f126fd19e9f8480069e991cc917a610fb0ab0)
+++ src/Entity/Client.php	(date 1543587705443)
@@ -90,7 +90,7 @@
     /**
      * @var string
      *
-     * @ORM\column(name="token", type="string", length=1, nullable=false)
+     * @ORM\column(name="token", type="string", length=40, nullable=false)
      */
     private $token;
 
